src/App.tsx

```diff
@@ -1,14 +1,41 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { TodoModal } from './components/TodoModal';
 
 export const App: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isModalLoading, setIsModalLoading] = useState(false);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>(todos);
+  const [isModalSeen, setIsModalSeen] = useState(false);
+
+  const [defaultTodo, setDefaultTodo] = useState({
+    title: '',
+    id: 0,
+    completed: false,
+    userId: 0,
+  });
```

Comment:
```suggestion
  const defaultTodo = {
    title: '',
    id: 0,
    completed: false,
    userId: 0,
  };

  const [defaultTodo, setDefaultTodo] = useState(defaultTodo);
```
You could move defaultTodo outside of component and then set it, so it's more readable.